src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/CreateCollectionOperation.java

```diff
@@ -225,8 +225,9 @@ TableMetadata findTableAndValidateLimits(
       throw new JsonApiException(
```

Comment:
Maybe use the new "ErrorCode.toApiException()" method to simplify code?

Reply:
Got it. I forget u did the cleanup for this!